### PR TITLE
Less pendant and verbose descriptions for marine uniforms.

### PR DESCRIPTION
**Pull request by Ghommie**
that, and little clean up. Split it off a #240 for atomization.
:cl:
spellcheck: Made military uniforms/fatigues descriptions less pedantically verbose.
code: Removes a snowflake check from marine uniforms sensors as the parent path already has it covered.
/:cl:

### CODE REVIEW

**Comment by Arachnidnexus** [comment]
The "robust-proof" bit could probably be replaced with the "for the corps" bit and I don't know how relevant the EMF part is given the lack of that in the game.

**Comment by Alterist** [comment]
someone really like the line he wrote and decided to put it everywhere, huh?

**Comment by Ghommie** [comment]
> 
> 
> The "robust-proof" bit could probably be replaced with the "for the corps" bit and I don't know how relevant the EMF part is given the lack of that in the game.

The description always struck me with a tv shop ad feeling, exaggerating on the little points of armor it offers per stat. So I opted to retain the EMF part and add in the "(not) robust-proof" lines.

File: code/modules/clothing/under/marine_uniform.dm [hunk 1]
```diff
@@ -5,11 +5,12 @@
 
 /obj/item/clothing/under/marine
 	name = "\improper TGMC uniform"
-	desc = "A standard-issue Marine uniform. It has shards of light Kevlar to help protect against stabbing weapons, bullets, and shrapnel from explosions, a small EMF distributor to help null energy-based weapons, and a hazmat chemical filter weave to ward off biological and radiation hazards."
+	desc = "A standard-issue, kevlar-weaved, hazmat-tested, EMF-augmented marine uniform. You suspect it's not as robust-proof as advertised."
 	siemens_coefficient = 0.9
 	icon_state = "marine_jumpsuit"
 	armor = list(melee = 5, bullet = 5, laser = 5, energy = 5, bomb = 5, bio = 5, rad = 5)
 	rollable_sleeves = TRUE
+	has_sensor = 2
```
Comment (DominikPanic) [inline code/modules/clothing/under/marine_uniform.dm]: Undocumented change.

Comment (Ghommie) [reply code/modules/clothing/under/marine_uniform.dm]: Documented now.

**Pull request merged**